furusiya/model/entities/party/player.py

```diff
@@ -71,7 +80,12 @@ def move_or_attack(self, dx, dy):
         else:
             self.move(dx, dy)
             if self.mounted:
-                self.move(dx, dy)
+                if self.mounted_moves >= 1:
```

Comment:
I don't understand what these counts are for. Maybe more verbose names will increase understandability?